docs/gettingstarted/query_test_report.rst

```diff
@@ -45,6 +45,17 @@ You may find it useful to fetch all failed records and determine pass/fail crite
 
 .. _report_format_fields:
 
+Detailed Reports (``buildtest report --detailed``)
+---------------------------------------------------
+
+The **buildtest report --detailed** option can be used to show a detailed test report that may be of interest when
+examining a test. This option is synonymous to running ``buildtest report --format name, id, user, state, returncode,
+runtime, outfile, errfile, buildspec``. Shown below is an example output.
+
+.. dropdown:: ``buildtest report --detailed``
+
+    .. command-output:: buildtest report --detailed
+
```

Comment:
After this section please document that --detailed and --format can't be used together they are mutually exclusive options that affect format fields.

You can have the command run with command-output and note this command with exit 1. See in other documentation example on how we document commands that fail.

Reply:
See https://buildtest.readthedocs.io/en/devel/gettingstarted/buildingtest.html#invalid-buildspecs